server/config/config.go

```diff
@@ -145,7 +145,7 @@ func DefaultConfig() *Config {
 			RPCMaxBodyBytes:    1000000,
 		},
 		GRPC: GRPCConfig{
-			Enable:  false,
+			Enable:  true,
```

Comment:
I think @alexanderbez didn't want enabled by default

Reply:
Mhhh I can't recall this, but in any case, I don't see any harm. Do you?

Reply:
I think it's fine 🤷🏻‍♂️. It's probably a better UX to enable by default